sematic/ui/packages/main/src/components/RunList.tsx

```diff
@@ -14,7 +14,7 @@ import { useFetchRunsFn } from "../hooks/pipelineHooks";
 import useLatest from "react-use/lib/useLatest";
 import usePreviousDistinct from "react-use/lib/usePreviousDistinct";
 import { styled } from "@mui/system";
-import { Run } from "src/Models";
+import { Run } from "../Models";
```

Comment:
Can you please keep the root-relative import path?

Reply:
I agree. It makes the file resilient to moves, and also easier to understand where things are for developers.

Reply:
Thanks for the feedback. I agree. 

I refactored a couple of files only in this PR to change to use absolute path. To avoid deviate the main focus of the PR, I will do the rest gradually in future works. 

BTW, FYI, there is a [VS Code setting](https://stackoverflow.com/questions/52432191/auto-import-in-visual-studio-code-only-offering-absolute-path-with-lerna-subpack) to determine whether to use relative or non-relative paths automatically.